build.gradle.kts

```diff
@@ -1,6 +1,9 @@
+import com.android.build.gradle.internal.tasks.factory.dependsOn
+
 buildscript {
     val kotlinVersion = "1.6.10"
     val sqlDelightVersion = "2.0.0-SNAPSHOT"
+    val dokkaVersion = "1.6.10"
```

Comment:
@gongracr It's not always doable.
Just like Coroutines _tries_  to keep up with Kotlin version, it normally isn't the exact same.
If there's a bug fix on Dokka, for example, they won't wait for the next version of the language to change it. Or if there's no breaking.

We switched today to kotlin `1.6.20-RC` today for example. Because it was breaking on Mac M1 due to usage of old version of NodeJS by Kotlin Multiplatform (JS).

And for a more concrete example. Dokka's `1.6.10` was released 2 weeks after Kotlin's `1.6.10`.

Reply:
Not doable.